iib/workers/tasks/build.py

```diff
@@ -1000,6 +1011,7 @@ def handle_add_request(
             prebuild_info['binary_image_resolved'],
             from_index,
             overwrite_from_index_token,
+            (prebuild_info['distribution_scope'] in ['dev', 'stage']),
```

Comment:
Can we add a test to verify that `_opm_index_add` is being called correctly given the distribution scope? If I change this line to just `False` no tests will fail.

Reply:
Yeah this is an oversight on my part, sorry.

Fixing now.